java/maven.indexer/src/org/netbeans/modules/maven/indexer/NexusRepositoryIndexerImpl.java

```diff
@@ -526,13 +526,13 @@ private void indexLoadedRepo(final RepositoryInfo repo, boolean updateLocal) thr
                     }
                     // MINDEXER-42: cannot use WagonHelper.getWagonResourceFetcher
                     Wagon wagon = embedder.lookup(Wagon.class, protocol);
-                    if (wagon instanceof HttpWagon) { //#216401
-                        HttpWagon httpwagon = (HttpWagon) wagon;
-                        //#215343
-                        Properties p = new Properties();
-                        p.setProperty("User-Agent", "netBeans/" + System.getProperty("netbeans.buildnumber"));
-                        httpwagon.setHttpHeaders(p);
-                    }
+//                    if (wagon instanceof HttpWagon) { //#216401
+//                        HttpWagon httpwagon = (HttpWagon) wagon;
+//                        //#215343
+//                        Properties p = new Properties();
+//                        p.setProperty("User-Agent", "netBeans/" + System.getProperty("netbeans.buildnumber"));
+//                        httpwagon.setHttpHeaders(p);
+//                    }
```

Comment:
todo: investigate build error
```
/home/mbien/NetBeansProjects/netbeans/java/maven.indexer/src/org/netbeans/modules/maven/indexer/NexusRepositoryIndexerImpl.java:529: error: cannot access AbstractHttpClientWagon
                    if (wagon instanceof HttpWagon) { //#216401
  class file for org.apache.maven.wagon.shared.http.AbstractHttpClientWagon not found
```

some class moved probably. Could be done via reflection since it is just a method and overall not so important.